p2p/p2p.go

```diff
@@ -121,18 +132,23 @@ func (s *Service) start(e chan error) {
 		e <- errors.New("no peers to bootstrap to")
 	}
 
-	err := s.dht.Bootstrap(s.ctx)
-	if err != nil {
-		e <- err
-	}
-
-	if !s.noBootstrap {
-		// connect to the bootstrap nodes
-		err := s.bootstrapConnect()
-		if err != nil {
-			e <- err
+	go func() {
```

Comment:
Afaik this can likely be removed once we can communicate back to the substrate nodes? Please document if this is the case.

Reply:
yes probably, I think we get kicked off after a few minutes due to inactive. I will document